background/score/activities.go

```diff
@@ -56,12 +62,27 @@ func (s *ScoreUpdateWorker) CalculatePOIStateActivity(ctx context.Context, id st
 	return &metric, nil
 }
 
+func (s *ScoreUpdateWorker) CheckLocationSpikeActivity(ctx context.Context, spikeSymptomTypes []schema.SymptomType) ([]schema.Symptom, error) {
```

Comment:
This function transfers symptom ID to Symptom struct. Maybe don't use the verb check in the function name.